DutchFilter/sections/specific.txt

```diff
@@ -4,6 +4,8 @@
 ! Good: example.org###rek; ||example.com/ads/
 ! Bad: example.org#@##banner_ad (for instance, should be in allowlist.txt or in antiadblock.txt)
 !
+voetbalzone.nl##div[class*="ad-wrapper"]
+voetbalzone.nl##div[class="fco-openweb-ad"]
```

Comment:
```suggestion
voetbalzone.nl##div[class^="ad-wrapper"]
voetbalzone.nl##.fco-openweb-ad
```
In general, rules are good, but 1st with `^` is preferably.
2nd - `.fco-openweb-ad` selector will work if other classes are present, for example
![image](https://github.com/user-attachments/assets/037072fa-8a76-4858-82e7-8fe3c8d7cc51)